docs/usage.md

```diff
@@ -0,0 +1,94 @@
+# Arcadia-Science/metagenomics Usage
+
+This page provides documentation on how to use the Arcadia-Science/metagenomics pipeline.
+
+## Input specifications
+
+This pipeline processes metagenomic reads in FASTQ format from either Illumina or Nanopore technologies. **Important note**: This pipeline separately processes Illumina or Nanopore metagenomes and does not perform hybrid assembly, polishing of Nanopore assemblies with Illumina reads, or scaffolding with Nanopore reads.
+
+For Nanopore reads, the pipeline takes as direct input a single fastq file and does not perform demultiplexing or basecalling, so it expects that those steps should be done prior to feeding into the pipeline and aggregating into a single fastq file.
+
+The workflow takes a CSV samplesheet listing the sample names and paths to the fastq files as direct input. It does not take the path to the directory of the fastq files, you must list them in a samplesheet. The CSV must contain the columns:
+
+`sample, fastq_1, fastq_2`
+Neither the `sample` or `fastq*` columns may contain spaces, and the `fastq` columns contains the path(s) to your fastq file(s).
```

Comment:
Why is that? Especially for `sample`?

Reply:
I guess it could be ok if sample had spaces, because names of files that incorporate pieces of the sample name come from the fastqs or downstream assemblies. I think it's a good rule of thumb though